src/main/java/controllers/PostPresenter.java

```diff
@@ -0,0 +1,9 @@
+package controllers;
+
+public class PostPresenter {
+
+    public void createPost() {
```

Comment:
The CRC card for PostPresenter says, "gets post creation related inputs (recipe steps, ingredients, category)". I think we could pass in those things into this method. Also, I think in the body of createPost, we would call PostManager. If PostManager exists in our project maybe call that and pass the parameters createPost got. Otherwise, maybe add a TODO to do that later, up to you